euphoria.cabal

```diff
@@ -1,131 +1,123 @@
--- Initial euphoria.cabal generated by cabal init. For further documentation,
--- see http://haskell.org/cabal/users-guide/
+cabal-version: 3.0
 
-name:                euphoria
-version:             0.8.0.0
-synopsis:            Dynamic network FRP with events and continuous values
+name: euphoria
+version: 1.0.0
```

Comment:
Don't update the version yet. We update it when we make a release.

Reply:
Fixed